### ISSUE TITLE
Lanenet speed on Jetson Xavier

### ISSUE THREAD

**Issue by ArtlyStyles** [MaybeShewill-CV/lanenet-lane-detection#415]
Hi, after converting my trained tensorflow mode the uff, I was able to run Lanenet with TensorRT on Jetson Xavier. Single 512x256 image inference time is 60ms (after run jetson_clcok) and I can process 17 frames per second. 

However, this speed is still not good enough for me. I am looking for 30-40 frames per second. I am thinking changing the image dimension. If I change the size to 640x100, could I still train the network and run inference?

**Comment by MaybeShewill-CV** [comment]
@ArtlyStyles You may test it yourself. The model can handle images with such image shape technologically but I did not test before. By the way did you use the model with bisenetv2 backbone or vgg backbone:)

**Comment by ArtlyStyles** [comment]
@MaybeShewill-CV I used vgg. Is bisenetv2 will be faster?

**Comment by ArtlyStyles** [comment]
I test bisenetv2, the detection results seems to be worse than VGG

**Comment by MaybeShewill-CV** [comment]
@ArtlyStyles BiseNet will be faster. The performance of bisenet based lanenet should be almost the same as which was based on vgg backbone:)

**Comment by ArtlyStyles** [comment]
I used to use a old version of lanenet (2019) and converted that to tensorRT. In that model, the output node names were "lanenet_model/vgg_backend/binary_seg/Softmax,lanenet_model/vgg_backend/instance_seg/pix_embedding_conv/pix_embedding_conv"

Yesterday, I download the latest lanenet from this report and used the bisenetv2 as backbone, but the following output node names are not correct:
"lanenet_model/bisenetv2_backend/binary_seg/Softmax,lanenet_model/bisenetv2_backend/instance_seg/pix_embedding_conv/pix_embedding_conv"

So what are the correct output node names in with bisenetv2?

**Comment by MaybeShewill-CV** [comment]
@ArtlyStyles You may print the graph node name and fetch any node you want. The output node should be which their name are similar with what you mentioned above:)

**Comment by ArtlyStyles** [comment]
Could you let me know how to print the names?

**Comment by MaybeShewill-CV** [comment]
@ArtlyStyles After saver restored model you may do something as follows:
          for vv in tf.global_vars(): print(vv.name) :)
